projects/views.py

```diff
@@ -31,21 +36,38 @@ def delete_project(request, proj_id):
     if request.method == "POST":
         proj.delete()
     return HttpResponseRedirect('/projects/list/')
-    #return render(request, 'projects/delete.html', {'proj': proj})
 
 @login_required(login_url=loginUser)
 def add_edit_project(request, proj_id = None):
     if proj_id:
         proj = get_object_or_404(Project, id = proj_id)
-        form = ProjectForm(request.POST or None, instance = proj)
+        print(request.FILES)
+        form = ProjectForm(request.POST or None, request.FILES if len(request.FILES)>0 else None, instance=proj)
     else:
-        form = ProjectForm(request.POST or None)
+        form = ProjectForm(request.POST or None, request.FILES if len(request.FILES)>0 else None)
     if request.method == "POST":
         if form.is_valid():
             form.save()
+            if form.is_valid():
```

Comment:
Potentially a minor nitpick. Is there a reason for the nested form.is_valid()? Seems like everything in the following block of code can simply be placed directly after the first form.save()